Add agent version API to agent reference (#2207)
sensu/sensu-docs: content/sensu-go/5.18/reference/agent.md

### Get service version

There are two ways to get the current agent version: the `sensu-agent` tool and the agent version API.

> To get the current agent version with the `sensu-agent` tool:

Reviewer: ```suggestion
To get the version of the current `sensu-agent` tool:
```

{{< highlight shell >}}
sensu-agent version
{{< /highlight >}}

> To get the current agent version with the agent version API:

Reviewer: ```suggestion
To get the version of the running `sensu-agent` service:
```
